extractor/src/test/java/org/schabi/newpipe/extractor/services/youtube/search/YoutubeMusicSearchExtractorTest.java

```diff
@@ -153,8 +153,8 @@ public static void setUp() throws Exception {
 
     public static class CorrectedSearch extends DefaultSearchExtractorTest {
         private static SearchExtractor extractor;
-        private static final String QUERY = "duo lipa";
-        private static final String EXPECTED_SUGGESTION = "dua lipa";
+        private static final String QUERY = "nocopyrigh sounds";
```

Comment:
Is this typo intentional? If yes, consider adding a comment.

Reply:
Yes, it is. look at the test name: "CorrectedSearch" and EXPECTED_SUGGESTION below. I don't think it needs a comment, the variable / class / function names says everything